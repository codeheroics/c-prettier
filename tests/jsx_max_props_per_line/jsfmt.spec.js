run_spec(__dirname, ["flow", "typescript"]);
run_spec(__dirname, ["flow", "typescript"], { jsxMaxPropsPerLine: 1 });
run_spec(__dirname, ["flow", "typescript"], { jsxMaxPropsPerLine: 2 });
