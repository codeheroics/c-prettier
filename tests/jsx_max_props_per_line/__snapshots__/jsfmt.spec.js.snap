// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test.js 1`] = `
<div attr1="aaaaaaaaaaaaaaaaa" attr2="bbbbbbbbbbb" />;
<div attr1="aaaaaaaaaaaaaaaaa" attr2="bbbbbbbbbbb" attr3="cccccccccccc" />;
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<div attr1="aaaaaaaaaaaaaaaaa" attr2="bbbbbbbbbbb" />;
<div attr1="aaaaaaaaaaaaaaaaa" attr2="bbbbbbbbbbb" attr3="cccccccccccc" />;

`;

exports[`test.js 2`] = `
<div attr1="aaaaaaaaaaaaaaaaa" attr2="bbbbbbbbbbb" />;
<div attr1="aaaaaaaaaaaaaaaaa" attr2="bbbbbbbbbbb" attr3="cccccccccccc" />;
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<div
  attr1="aaaaaaaaaaaaaaaaa"
  attr2="bbbbbbbbbbb"
/>;
<div
  attr1="aaaaaaaaaaaaaaaaa"
  attr2="bbbbbbbbbbb"
  attr3="cccccccccccc"
/>;

`;

exports[`test.js 3`] = `
<div attr1="aaaaaaaaaaaaaaaaa" attr2="bbbbbbbbbbb" />;
<div attr1="aaaaaaaaaaaaaaaaa" attr2="bbbbbbbbbbb" attr3="cccccccccccc" />;
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<div attr1="aaaaaaaaaaaaaaaaa" attr2="bbbbbbbbbbb" />;
<div
  attr1="aaaaaaaaaaaaaaaaa"
  attr2="bbbbbbbbbbb"
  attr3="cccccccccccc"
/>;

`;
